frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1343,6 +1356,10 @@ function ReaderHighlight:deleteHighlight(page, i, bookmark_item)
     self.ui:handleEvent(Event:new("DelHighlight"))
     logger.dbg("delete highlight", page, i)
     local removed = table.remove(self.view.highlight.saved[page], i)
+    -- Clear the hash if there aren't any more highlights on this page
```

Comment:
Clear the "hash" ? Do you mean "table" ?

Reply:
Yeah, that was my vague 4AM wording attempt at making it clear the it's indexed by hash, despite those hashes being plain ints, to explain why the nested items are table.removed because *those* are pure arrays, but not the outer one ^^.